front/lib/api/assistant/actions/browse.ts

```diff
@@ -314,6 +315,7 @@ export class BrowseConfigurationServerRunner extends BaseActionConfigurationServ
       type: "browse_success",
       created: Date.now(),
       configurationId: agentConfiguration.sId,
+      runId: await dustRunId,
```

Comment:
I dont think we want to expose the run ids in the event since we really don't need it anywhere.

Let's keep this private to each action